version1/02 - Object types.md

```diff
@@ -757,6 +759,30 @@ The following notation is used in the tables below:
+| M601      | string    | opprettetAvBrukerIdent |    x     |      x      |                  |  |  cw  |  x   |   x   |  x   |      |        |                                                                                                                                |
+| M020      | string    | title                  |    x     |      x      |                  |  |  x   |  x   |   x   |  x   |      |        |                                                                                                                                |
+| M021      | string    | beskrivelse            |          |             |                  |  |  x   |  x   |   x   |  x   |      |        |                                                                                                                                |
+| M111      | date      | presedensDato          |    x     |      x      |                  |  |  x   |  x   |   x   |  x   |      |        |                                                                                                                                |
+| M628      | timestamp | presedensGodkjentDato  |    x     |      x      |                  |  |  x   |  x   |   x   |  x   |      |        |                                                                                                                                |
+| M629      | string    | presedensGodkjentAv    |    x     |      x      |                  |  |  x   |  x   |   x   |  x   |      |        |                                                                                                                                |
+|           |           |                        |          |             |                  |  |      |      |       |      |      |        |                                                                                                                                |
+| M056      | string    | presedensStatus        |          |      x      | presedensStatus  |  |  x   |  x   |   x   |      |      |        |                                                                                                                                |
+| M311      | string    | presedensHjemmel       |          |             | presedensHjemmel |  |  x   |  x   |   x   |      |      |        |                                                                                                                                |
+| M312      | string    | rettskildefaktor       |          |             | rettskildefaktor |  |  x   |  x   |   x   |      |      |        |                                                                                                                                |
```

Comment:
Should have a mark for not null.

Reply:
Addressed with https://github.com/documaster/noark5-web-services/pull/50/commits/3e7fa81e3c1996d2a7488b335d719518ca15bed6